Add Swedish translation

MorganAntonsson opened the pull request:
Hi, I have created a Swedish translation. I also added missing translations to the Makefile.

p91paul commented:
sorry I forgot to update pot file: please look at the "it" translation, you should find one additional string which needs translation at the end.
I will regenerate pot ASAP.

MorganAntonsson commented:
Thanks for the feedback. I have added the missing string. I also added a "mergepo" target in the Makefile to make it easier to update the .po files and simplified the "translations" rule. I hope that's OK?

p91paul commented:
Thanks a lot!
